pkg/config/config.go

```diff
@@ -122,9 +122,16 @@ in a user friendly format`, binName, binName)
 	flag.StringVar(&userConfig.Profile, "profile", "", "[OPTIONAL] AWS CLI Profile to use for credentials and config")
 	flag.StringVar(&userConfig.Region, "region", "", "[OPTIONAL] AWS Region to use for API requests")
 	flag.StringVar(&userConfig.Bucket, "bucket", "", "[OPTIONAL] the name of the Bucket created in the last run. When provided with this flag, the CLI won't create new resources, but try to grab test results from the Bucket. If you provide this flag, you don't need to specify any required flags")
+	flag.BoolVar(&userConfig.IsDemo, "demo", false, "[OPTIONAL] set to true if you'd like the tool to execute against the demo app, app-for-e2e")
```

Comment:
That makes sense. I'm removing `--demo` and relocating the demo template file in this PR.

For the script that sets everything up, I'll need to dig deeper into how we can pass the `master_demo` template and/or additional roles to be appended to the existing template since we're not looking to have demo-specific code in the application. This will be addressed in next PR

Reply:
I'd probably recommend setting up that infrastructure through  a separate cloudformation stack outside of qualifier.

Reply:
Could you clarify what infrastructure exactly? The instances to run the demo will be provisioned by IQ and these instances will need permissions to access the DynamoDB table. Maybe I'm misunderstanding, but even if we can edit the instance permissions outside of IQ I think that adds more confusion unless we are advising users to do the same

Reply:
I think giving permissions to the instance would be a common use case of an end user.  So probably makes sense to accept an IAM role for a qualification run (easier) or allow the qualifier agent to be installed on instances that an end user provisions through cfn with their regular app (probably much harder right now).

Reply:
agreed on passing additional roles/policies. I need to do a deep dive on that most likely another PR given the code changes.

I've added asana tasks for both (I think separating agent out could be good in the future, but not top priority)